pep-0594.rst

```diff
@@ -848,17 +528,6 @@ It also required a configure flag that caused an ABI incompatibility. The
 module was removed in 3.7 by Nathaniel J. Smith in
 `bpo-29137 <https://bugs.python.org/issue29137>`_.
 
-Module type
-  C extension + CAPI
-Deprecated in
-  3.7
-Removed in
-  3.7
-Has a designated expert
-   n/a
-Substitute
-  **none**
-
 
 Modules to keep
 ===============
```

Comment:
I suggest removing `optparse` as a replacement for `getopt`; you could consider adding Click to both as another replacement (which uses a private fork of `optparse` under the hook to provide its features/behavior). You could also add [pyupgrade](https://github.com/asottile/pyupgrade) as a possible replacement for `lib2to3`. I also suggest removing `colour`, as it has [no commits since 2017](https://github.com/vaab/colour) and relies on a bunch of deprecated/EoL packaging stuff (EOL d21, horrific setup.py hacks, etc).

Reply:
The key point of the suggestions isn't necessarily the _best_ option, but to prove the point that there _is_ a serviceable option. That means optparse is a legitimate replacement for getopt.

Reply:
Understood, thanks.